README.md

```diff
@@ -18,6 +18,59 @@ Read ["Installing Rust"] from [The Book].
 
 ## Building from Source
 
+Starting from `e645ed59` the Rust Build System is being converted to use
```

Comment:
Before we get experience with using Cargo I don't think we want to mention it so prominently in the README. I suspect there will be a long tail of many minor bugs to fix, and we can always add this info later on down the road.